sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -336,23 +340,33 @@ export class OfferingValidationModel {
   @Min(0, {
     message: getMinFormatMessage(userFriendlyNames.actualTuitionCosts),
   })
-  @Max(MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE, {
+  @IsNumber(currencyNumberOptions, {
```

Comment:
What about having a single decorator which checks everything related to offering costs?

Reply:
Yes custom decorator. I see your point, but as already we have pitched in customization here with `@valid` with custom constraint class, I felt we can go for custom decorator.

Reply:
@ann-aot did you try adding the @Max twice maybe using a different groupName in the options?

Reply:
As discussed with @andrewsignori-aot , added a decorator function to the custom validator, https://github.com/bcgov/SIMS/pull/2529#discussion_r1406884673